invenio_vocabularies/alembic/4a9a4fd235f8_create_vocabulary_schemes.py

```diff
@@ -20,10 +20,10 @@ def upgrade():
     """Upgrade database."""
     op.create_table(
         "vocabularies_schemes",
-        sa.Column("id", sa.String(), nullable=False),
-        sa.Column("parent_id", sa.String(), nullable=False),
-        sa.Column("name", sa.String(), nullable=True),
-        sa.Column("uri", sa.String(), nullable=True),
+        sa.Column("id", sa.String(255), nullable=False),
```

Comment:
Curious comment (no need to take any action), sharing in case someone wondered as I did about:

> Will this add padding and consume space (🤔?), for example this id is usually a recid v2 which has 8 charaters plus a hypen (`_`) for a total of 9.

Answer: Seems it is safe to just put 255. References: [DB StackExchange](https://dba.stackexchange.com/questions/125499/what-is-the-overhead-for-varcharn), [PosgreSQL docs](https://www.postgresql.org/docs/current/datatype-character.html), [MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/char.html). Spaces are ignored so they are represented as `''` and consume 1 byte.